libcudacxx/include/cuda/std/detail/libcxx/include/complex

```diff
@@ -295,8 +295,53 @@ template<class T> complex<T> tanh (const complex<T>&);
 
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
+template <class _Tp>
+struct __is_nvfp16 : false_type
+{};
+
+template <class _Tp>
+struct __is_nvbf16 : false_type
+{};
+
+template <class _Tp>
+struct __is_complex_float
```

Comment:
I am wondering whether we should name that rather `__is_extended_floating_point`

Reply:
I'll move that around in the PR I'll eventually do for atomic, though I take issue with that name, because these are *not* extended fp types per the standard definition.